src/main/scala/beam/router/skim/readonly/TransitCrowdingSkims.scala

```diff
@@ -87,7 +87,7 @@ class TransitCrowdingSkims(vehicleTypes: Map[Id[BeamVehicleType], BeamVehicleTyp
     def getValueFrom(x: collection.Map[AbstractSkimmerKey, AbstractSkimmerInternal]) = {
       x.get(key).asInstanceOf[Option[TransitCrowdingSkimmerInternal]]
     }
-
+    numberOfRequests = numberOfRequests + 1
```

Comment:
Let's actually calculate the rest of statistics for this skimmer too, let's assume each time either a value in aggregatedFromPastSkims or in the pastSkims found we increase numberOfSkimValueFound, otherwise let's assume it is a default value returned